src/renderer/screens/ColorPalette/ColorButtonsArea.js

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
Same here, we have copyright in every file (as far as I remember), new ones should have them as well. You can copy & paste from any of the existing files and change the copyright holder to your name (or leave it at keyboardio, if you're ok with that).